### ISSUE TITLE
number of symbols in prelude: 0

### ISSUE THREAD

**Issue by Alok** [eagletmt/neco-ghc#67]
results of `NecoGhcDiagnostics`:

```
Current filetype: haskell
ghc-mod is executable: 1
omnifunc: necoghc#omnifunc
neocomplete.vim: 0
neocomplcache.vim: 0
YouCompleteMe: 0
vimproc.vim: 902
ghc-mod: 5.6.0.0
Imported modules: Prelude
Number of symbols in Prelude: 0
```

I'm running latest neovim with latest deoplete and neco-ghc.

**Comment by Shougo** [comment]
I cannot reproduce the problem.
But this feature is asynchronous.

Your information is very poor.
We have not ESP skill.
I think you can debug `s:ghc_mod_caching_browse()` in autoload/necoghc.vim.

**Comment by sulami** [comment]
I did experience the same issue and dug around a lot. I found out that it only didn't work inside stack projects (created with `stack new`). After some more research, I downgraded my stack resolver from 7.9 (using GHC 8) to 6.25 (using GHC 7.10), which fixed this issue. This seems to be related to this issue: https://github.com/DanielG/ghc-mod/issues/772. Did not look much further, I just assume some part of the chain isn't ready for GHC 8 yet.
